### PR TITLE
added indeed job scraper files and reqs

### PR DESCRIPTION
**Pull request by qdupupet**

### CODE REVIEW

File: services/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+numpy==1.19.1
+pandas==1.1.1
+python-dateutil==2.8.1
+pytz==2020.1
+PyYAML==3.13
+requests==2.22.0
+six==1.15.0
+soupsieve==2.0.1
+urllib3==1.25.10
+us==1.0.0
```
Comment (mandeldr) [inline services/requirements.txt]: We should combine the requirements.txt into a single file. Also I would test out the functions once you do that so we know it integrates properly. Also which function should I be calling from the form?

**Review by mandeldr — COMMENTED** [review]
Just left a couple of comments and questions.

File: services/requirements.txt (hunk 1 above)
Comment (katiehouse) [reply services/requirements.txt]: You will call `scrape_indeed()` in the main.py file

**Comment by katiehouse** [comment]
@qdupupet @mandeldr I just updated the requirements.txt 👍

**Pull request merged**